CHANGES_NEXT_RELEASE

```diff
@@ -10,3 +10,6 @@
   * sinon-chai
   * grunt and grunt related module
   * closure-linter-wrapper
+- Change on the PERSEO_ORION_URL env var behaviour. Now it represents Context Broker URL instead of the updateContext
+  endpoint
+- Add: NGSIv2 support
```

Comment:
I'd suggest to be more precise here. I mean, something like this:

```
Add: NGSIv2 support in both notification reception and CB update action
```

In addition, I'd suggest to mention the added dependenciy (ngsijs) and remove chai, sinon and sinon-chai from the list of removed dependencies ;) In fact, if the re-introduced version is different from the old one a line line this one "Upgrade dev dependency chai|sinon|sinon-chai from ... to ..." should be added.

Reply:
Fixed in 330b1b0